src/editors/containers/EditorContainer/index.tsx

```diff
@@ -42,7 +42,13 @@ const OuterWrapper: React.FC<WrapperProps> = ({ fullScreen, children }) => {
   );
```

Comment:
The line just above here throws a warning because some "required" props (`onClose` and `title`) from Paragon are missing.
```jsx
    <ModalDialog isOpen size="xl" isOverflowVisible={false} onClose={() => {}} title="">{children}</ModalDialog>
```

Or maybe it is better just to fix it in Paragon instead.

Reply:
Ah, thanks! That's why the ESC key wasn't working. Fixed :)